modules/topicsFpdModule.js

```diff
@@ -86,10 +90,14 @@ export function getTopicsData(name, topics, taxonomies = TAXONOMIES) {
     );
 }
 
+function isTopicsSupported(doc = document) {
+  return 'browsingTopics' in doc && doc.featurePolicy.allowsFeature('browsing-topics')
```

Comment:
is it guaranteed `allowsFeature` will always be there if browsingTopics is? So no need to safe check here?